src/js/components/ClusterDropdown.d.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactNode, Component } from "react";
+
+interface Props {}
+
+interface State {
```

Comment:
🎨 I think we used to `ComponentNameProps` and should stickt to one pattern… But thats nothing blocking here

Reply:
Can you explain why? I didn't understand the pattern in the first place

Reply:
I think its the same as having `JobTypeResolver` vs `TypeResolver` and `hostnameRenderer` vs `renderer` 🤷‍♂️ 

Personally, I like the more verbose thing more - but if there is a reason to use the shorter ones (e.g. for parsing) - I'm happy change my opinion 👍